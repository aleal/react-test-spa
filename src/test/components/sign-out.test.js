import React from 'react';
import {renderComponent} from '../test-helper';
import SignOut from '../../components/sign-out';

describe('Sign Out Tests' , () => {
    
    let component;

    beforeEach(() => {
        component = renderComponent(SignOut);
    });
  
    it('renders sign out page', () => {
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});