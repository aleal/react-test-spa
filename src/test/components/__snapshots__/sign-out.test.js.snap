// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign Out Tests renders sign out page 1`] = `
<div
  className="container"
>
  <h2>
     Sign out 
  </h2>
  <p>
     You has been signed out. See you soon!!! 
  </p>
</div>
`;
